ConnectApiHelper.cls

```diff
@@ -234,7 +243,14 @@ global class ConnectApiHelper {
         mentionSegment.id = mentionId;
         return mentionSegment;
     }
+
+    private static ConnectApi.EntityLinkSegmentInput makeEntityLinkSegmentInput(String recordId,String altText) {
```

Comment:
You can remove the `altText` parameter since it's not used. Then you can call it on L178 with `makeEntityLinkSegmentInput(recordInfo[1])` instead of `makeEntityLinkSegmentInput(recordInfo[1], null)`.